pkg/gui/gui_config.go

```diff
@@ -0,0 +1,15 @@
+package gui
+
+import gui_server "github.com/Kong/kuma/pkg/config/gui-server"
+
+type GuiConfig struct {
+	ApiUrl      string `json:"apiUrl"`
+	Environment string `json:"environment"`
+}
+
+func FromServerConfig(config gui_server.GuiConfig) GuiConfig {
+	return GuiConfig{
+		ApiUrl:      config.ApiUrl,
+		Environment: config.Environment,
```

Comment:
Similarly to the earlier comment, why do we configure `Environment ` separately leaving a room for a contradiction between `ApiUrl` and `Environment`  ?